config/routes.rb

```diff
@@ -131,6 +131,11 @@
     get '/sms/submit' => 'sms#create'
```

Comment:
Can we move the SMS routes into a separate scope block, then your comment below can apply to the whole ODK scope block.

The SMS stuff is going to change soon too, with the token ticket that is upcoming.

Also we could perhaps put direct_auth: true in the defaults for the whole block to simplify things. The condition in authentication.rb would still work even for the the no_auth one. We could consider no_auth a form of direct_auth, if you catch my drift.